modules/idLibrary.js

```diff
@@ -203,7 +203,7 @@ function postData() {
   logInfo('Users' + JSON.stringify(userIds));
```

Comment:
can we use `userIds` here as well?

Reply:
Updated from JSON.stringify(userIds) to userIds